README.md

```diff
@@ -16,19 +18,22 @@ go get -u github.com/ncr4/tuneuptechnology-go
 package main
 
 import (
+	"fmt"
+	"os"
+	"encoding/json"
 	"github.com/ncr4/tuneuptechnology-go"
 )
 
 func main() {
-	tuneuptechnology.CreateCustomer(
-		// Pass in your email and API key
-		&tuneuptechnology.Client{
-			Auth: "",
-			APIKey: "",
-		},
+	// Setup your email and API key
+	api_email := "EMAIL_HERE"
+	api_key := "API_KEY_HERE"
 
-		// Create a customer passing in all required params
+	// Create a customer passing in all required params
+	customer := tuneuptechnology.CreateCustomer(
 		&tuneuptechnology.Customer{
+			Auth: api_email,
+			APIKey: api_key,
 			Firstname: "Go",
 			Lastname: "Test",
 			Email: "go-test@test.com",
```

Comment:
Use `@example.com` emails to avoid spamming any live domains

Reply:
I'm not sure I follow, these are just variables here for whatever the user put above.

Reply:
Scrapers and whatnot are trawling through github all the time. You want to avoid sending those crawlers and bots to real domains, so using a standard dummy one like `example.com` is preferred, since it is set up to handle (and reject) the incoming traffic and emails.

Reply:
Ah I didn't see the couple of lines below it haha. Updated.